cmd/agent/main.go

```diff
@@ -158,7 +158,7 @@ func run(ctx context.Context) error {
 				os.Exit(1)
 			}
 			// onConnect will use the transport later on, so discard it as it doesn't work and fallback to the system store.
-			transport = nil
+			transport = nil //nolint:ineffassign
```

Comment:
@Oats87 Can we check if this a false positive or it needs to be fixed?

Reply:
This line can be removed. The code which used it was removed along with the RKE1 removal (see: https://github.com/jakefhyde/rancher/commit/05434e483475ae05cf8c5de068d50536ddce06e8#diff-61358711e980ccf505246fd3915f97cbd3a380e9b66f6fa5aad46749968c5ca3L346-L348)

Reply:
Done in [006bc7a](https://github.com/rancher/rancher/pull/53045/commits/006bc7a9130bf3db245799ee0f3b4a5e8f049d1d)